src/Core/IEPlugin/#IEPluginInit.cc#

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
@kgillette One last thing: please delete this temporary file that was accidentally committed.